### ISSUE TITLE
Header box after TOC selection

### ISSUE THREAD

**Issue by BoPeng** [tscanlin/tocbot#121]
Direct going to this address is ok

https://vatlab.github.io/sos-docs/doc/user_guide/sos_in_notebook_tocbot.html#Execute-embedded-workflows-with-command-sos

but clicking on the TOC displays a blue box around the header:

![image](https://user-images.githubusercontent.com/9889312/51434805-b73fdd00-1c2e-11e9-99a2-3f6dd41bf3be.png)

Is this the effect some tocbot css?

**Comment by BoPeng** [comment]
More observations: clicking the header does not do anything initially, but clicking the header will show a blue frame after the header has been scrolled to by clicking its entry in the TOC, so the smooth scroll action has changed something of the headers, which is not good.

**Comment by tscanlin** [comment]
This is not the behavior of tocbot, this is native browser CSS resulting from a focus / active state.

See: https://stackoverflow.com/questions/20340138/remove-blue-border-from-css-custom-styled-button-in-chrome

**Comment by BoPeng** [comment]
Do you have to active/focus a header when you scroll to it？

**Comment by tscanlin** [comment]
Yes, that is the default browser behavior when the user clicks a link with a hash matching an element id on the page. You could write custom CSS to avoid the extra styling though.

**Comment by BoPeng** [comment]
I do not get it. Regular scrolling does not cause this because no frame will appear with scrollSmooth turned off.

**Comment by BoPeng** [comment]
As suggested in https://stackoverflow.com/questions/20340138/remove-blue-border-from-css-custom-styled-button-in-chrome, the following css solves the problem:

```
h1:focus, h2:focus, h3:focus, h4:focus, h5:focus {
  outline:none;
  box-shadow: none;
}
```

This obviously cannot go in `tocbot.css` so sooner or later other users will be bitten by this, so I think the focus behavior should be disabled, at least turned off by default.

**Comment by PaulRivier** [comment]
Hi, 
I will add my voice to this, tocbot should behave more intuitively by default, and avoid focusing element. Cheers.
